metricflow-semantics/metricflow_semantics/model/semantics/linkable_element.py

```diff
@@ -336,10 +336,6 @@ def __post_init__(self) -> None:  # noqa: D105
             self.derived_from_semantic_models
         ), "There must be at least one semantic model from which the metric is derived."
 
-        assert (
```

Comment:
Do we want to consider a logger.warning or something (debug is fine, anything that isn't an error) for tracking while we roll out other validation and see if this issue is what we think it is?

Reply:
Update - based on what we just learned, no, I don't think we want logging here, at least not at this time.